SelfDriveEnv.py

```diff
@@ -268,6 +320,32 @@ def render(self):
         pygame.display.flip()
         self.handle_events()
     
+    def calc_pos(self):
```

Comment:
Could you please leave some documentation with this function to help with #1 
Also, I think a generic name like `calc_pos` might get confusing in the long run. What about something like `calc_starting_pos`?

Reply:
I had that earlier but then the function also calculates the end position. I'm not sure If we need the end position though? I'll make it a separate function for possible future use then so it could be separated into more specific functions.

Reply:
Good idea. See my other comment for inspiration :)